fw-child/template/apps/query.php

```diff
@@ -21,39 +21,36 @@
 		
 	?>
 	
-	<div class="col-13 offset-1">
-		<div class="tab-drawer-bumper">
-			<div class="pt-6">
-				
-				<ul id="sort-menu" class="fw-query-sort list-unstyled d-flex all-caps">
-					<li class="selected me-4" data-sort="date_desc"><i class="fas fa-arrow-up me-2"></i> <?php _e ( 'Newest First', 'cdc' ); ?></li>
-					<li class="" data-sort="date_asc"><i class="fas fa-arrow-down me-2"></i> <?php _e ( 'Oldest First', 'cdc' ); ?></li>
-				</ul>
-				
-			</div>
+	<div class="col-12 offset-1 col-sm-13">
+		<div class="pt-6">
 			
-			<?php
-				
-				$module_args = array (
-					'posts_per_page' => 12,
-					'post_type' => 'app',
-					'post_parent' => 0,
-					'post_status' => 'publish'
-				);
-				
-			?>
+			<ul id="sort-menu" class="fw-query-sort list-unstyled d-flex all-caps">
+				<li class="selected me-4" data-sort="date_desc"><i class="fas fa-arrow-up me-2"></i> <?php _e ( 'Newest First', 'cdc' ); ?></li>
+				<li class="" data-sort="date_asc"><i class="fas fa-arrow-down me-2"></i> <?php _e ( 'Oldest First', 'cdc' ); ?></li>
+			</ul>
 			
-			<div id="apps-grid" class="py-6" data-args='<?php echo json_encode ( $module_args ); ?>'>
-				
-				<div id="" class="query-container ">
-					<div class="fw-query-items row row-cols-1 row-cols-sm-2 row-cols-md-3 g-3 g-lg-6" data-options='<?php echo json_encode ( $item_options ); ?>'>
-						
-						<div class="fw-query-item"></div>
+		</div>
+		
+		<?php
+			
+			$module_args = array (
+				'posts_per_page' => 12,
+				'post_type' => 'app',
+				'post_parent' => 0,
+				'post_status' => 'publish'
+			);
+			
+		?>
+		
+		<div id="apps-grid" class="py-6" data-args='<?php echo json_encode ( $module_args ); ?>'>
+			
+			<div id="" class="query-container ">
```

Comment:
```suggestion
			<div class="query-container">
```

Reply:
This has been taken car of in the ticket regarding the Apps page.